src/bindings.cpp

```diff
@@ -39,8 +39,12 @@ void start_v8_isolate(void *dll){
     v8::V8::InitializeICUDefaultLocation(V8_ICU_DATA_PATH);
   }
 #endif
+#if (V8_MAJOR_VERSION * 100 + V8_MINOR_VERSION) >= 704
```

Comment:
I fear that either or both of V8_BUILD_NUMBER and V8_PATCH_LEVEL are required as well due to the development process of v8. https://github.com/v8/v8/commit/ffee558e14e28fc8b1f9a3c10ea3615e0d686c7b#diff-13532dcb40ea4b69edc4fea8b217fcec introduced 'NewDefaultPlatform()'

Reply:
I'd rather condition on the version when the current api was removed (hence 7.4.xx)